apps/src/javalab/Playground.js

```diff
@@ -116,6 +121,11 @@ export default class Playground {
       // can't play sound if game is over
       return;
     }
+
+    const filename = soundData.filename;
+
+    const audioElement = this.getAudioElement();
+    this.setMediaElement(audioElement, filename);
```

Comment:
nit: could simplify to `this.setMediaElement(this.getAudioElement(), soundData.filename))`